tests/testsuite/fixture.go

```diff
@@ -103,7 +103,6 @@ func SynchronizedBeforeTestSetup() []byte {
 func BeforeTestSuiteSetup(_ []byte) {
 
 	worker := GinkgoParallelProcess()
-	rand.Seed(int64(worker))
```

Comment:
Is this deprecated?  `"k8s.io/apimachinery/pkg/util/rand"`

Reply:
reverted the patch here